ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -226,7 +234,10 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   }
 
   const legendWidth = legend && legend.position === 'Right' ? 200 : adjustedContentDimensions.width;
-  const legendHeight = legend && legend.position === 'Right' ? adjustedContentDimensions.height : 40;
+  let legendHeight = legend && legend.position === 'Right' ? adjustedContentDimensions.height : LEGEND_HEIGHT_SM;
+  if (adjustedContentDimensions.height > PANEL_HEIGHT_LG_BREAKPOINT) {
+    legendHeight = LEGEND_HEIGHT_LG;
+  }
```

Comment:
Should this check that the legend is positioned on the `Bottom`? (I'm noticing that in the line above, if the legend is positioned to the 'Right', we make it as tall as the content)

Reply:
I'm guessing that the intention is something like:

```
if (legend && legend.position === 'Right') {
  legendHeight = adjustContentDimensions.height
} else {
  legendHeight = adjustedContentDimensions.height > BREAKPOINT ? LEGEND_HEIGHT_LG : LEGEND_HEIGHT_SM
}